examples/scattered_energy.py

```diff
@@ -1,20 +1,44 @@
+from surface_potential_analysis.state_vector.state_vector_list import get_state_vector
+
 from coherent_rates.plot import (
-    plot_scattered_energy_change_comparison,
+    plot_occupation_against_energy_comparison,
+    plot_scattered_energy_change_state,
+    plot_thermal_scattered_energy_change_comparison,
 )
 from coherent_rates.system import (
     HYDROGEN_NICKEL_SYSTEM_1D,
     PeriodicSystemConfig,
+    get_hamiltonian,
 )
 
-# Generates a plot of energy change due to scattering against square of scattered
-# momentum for both free and bound particles
-
 if __name__ == "__main__":
     config = PeriodicSystemConfig((20,), (50,), 50, temperature=155)
     system = HYDROGEN_NICKEL_SYSTEM_1D
 
+    plot_occupation_against_energy_comparison(system, config, 1000, (150,))
```

Comment:
plot_occupation_against_energy_comparison(system, config, 3, (50,))
Is probably amore realistic scenario